src/plugins/binaryview.rs

```diff
@@ -347,6 +353,18 @@ pub fn view_contents(
     Ok(())
 }
 
+#[cfg(not(feature = "image"))]
+#[allow(unused)]
```

Comment:
Why this? If it is unused, can you please remove it.

Reply:
It's not the function that's unused but its argument, `lores_mode`. I totally forgot for a moment that I can just rename it, will do so in a moment.